pkg/JuliaInterface/src/JuliaInterface.c

```diff
@@ -90,10 +95,14 @@ int is_gapobj(jl_value_t * v)
 static Obj Func_NewJuliaCFunc(Obj self, Obj julia_function_ptr, Obj arg_names)
 {
     if (!IS_JULIA_OBJ(julia_function_ptr)) {
-        ErrorMayQuit("NewJuliaCFunc: <ptr> must be a Julia object", 0, 0);
+        SYNC_GAP {
+            ErrorMayQuit("NewJuliaCFunc: <ptr> must be a Julia object", 0, 0);
+        }
     }
     if (!IS_PLIST(arg_names)) {
-        ErrorMayQuit("NewJuliaCFunc: <arg_names> must be plain list", 0, 0);
+        SYNC_GAP {
```

Comment:
oh, and let's get rid of this `SYNC_GAP` and only use `BEGIN_GAP_SYNC()` (or vice versa) -- that way, I don't have to remember the meaning of three different macros, just two (or one).